src/main/java/service/UserDetailService.java

```diff
@@ -0,0 +1,40 @@
+package service;
+
+import entity.User;
+import org.springframework.security.core.authority.SimpleGrantedAuthority;
+import org.springframework.security.core.userdetails.UserDetails;
+import org.springframework.security.core.userdetails.UsernameNotFoundException;
+import org.springframework.stereotype.Service;
+import repository.UserRepository;
+
+import java.util.Collections;
+import java.util.Optional;
+
+
+@Service
+public class UserDetailService implements org.springframework.security.core.userdetails.UserDetailsService {
```

Comment:
Rename `UserDetailService` to `CustomUserDetailService` and get rid of the specified package `org.springframework.security.core.userdetails.`